tools/wptserve/wptserve/config.py

```diff
@@ -28,11 +28,25 @@ def _merge_dict(base_dict, override_dict):
 
 
 class Config(Mapping):
-    """wptserve config
+    """wptserve configuration data
 
-    Inherits from Mapping for backwards compatibility with the old dict-based config"""
-    def __init__(self, logger_name, data):
-        self.__dict__["_logger_name"] = logger_name
+    Immutable configuration that's safe to be passed between processes.
+
+    Inherits from Mapping for backwards compatibility with the old dict-based config
```

Comment:
In other words, to support code that uses `config["key"]`, right? Is there any new way too, or actually just the old way?

Reply:
So my question was if the "old dict-based config" is retained here because there are call sites to it, or if it just falls out of a reasonable way to implement this? The comment makes me wonder where the badness is. Not that all old things are bad, but in code comments it's usually disparaging :)

Reply:
IIRC, there's various handlers that read from it as if it were a dict. Ultimately, the cost of supporting both is very small.